### PR TITLE
Show a splash screen when start the app

### PR DESCRIPTION
**Pull request by OwenRamire**
screen-20200702-215928_2

When this app is starting, we are going to see a splash screen of the app icon

### CODE REVIEW

**Comment by xErikPsx** [comment]
Prueba

**Comment by xErikPsx** [comment]
Prueba

File: OwenConferenciasApp/screens/formsLogin/MainLoginLayout.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import {View, Text, Image, TouchableOpacity, StyleSheet} from 'react-native';
+import {NavigationContainer} from '@react-navigation/native';
+import {createStackNavigator} from '@react-navigation/stack';
+import SignIn from './SignInLayout';
+import SignUp from './SignUpLayout';
+import App from '../../App';
+
+function MainForm({navigation}) {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' , backgroundColor:'#F0F1F6'}}>
```
Comment (Loored) [inline OwenConferenciasApp/screens/formsLogin/MainLoginLayout.js:11]: I suggest to wrap it in a class so it makes the code easy to read. 😄

**Review by Loored — CHANGES_REQUESTED** [review]

File: OwenConferenciasApp/screens/ProfileUserLayout.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {View, Image, TouchableOpacity, Text, StyleSheet, Alert} from 'react-native';
```
Comment (Loored) [inline OwenConferenciasApp/screens/ProfileUserLayout.js:2]: Here you are calling the Alert component and not using it. Let's call only the components we need. 🙏

File: OwenConferenciasApp/screens/ProfileUserLayout.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            style={styles.userImg}
+          />
+          <View style={styles.userInfo}>
+            <Text style={styles.user}>User: owen ramirez</Text>
+            <Text style={styles.user}>Email: owenomar@example.com</Text>
+          </View>
+        </View>
+        <View style={styles.btnEditProfile}>
+          <TouchableOpacity>
+            <Text style={{color: 'white', fontWeight: 'bold'}}>Edit Profile</Text>
```
Comment (Loored) [inline OwenConferenciasApp/screens/ProfileUserLayout.js:20]: You can wrap it in a class for this text. It just minimal details! 😄

**Review by Loored — CHANGES_REQUESTED** [review]

**Review by Loored — APPROVED** [review]
Well done! 💯

**Review by JCarlosRom — APPROVED** [review]
Everything looks good

**Review by luisarmandodv — APPROVED** [review]

**Pull request merged**